src/__tests__/html-text-input-test.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import { HTMLInputView } from '../components/html-text-input.js';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+// Utility to find the HTML TextInput from the wrapper
+const findHTMLTextInputFromWrapper = ( wrapper ) => {
+	const placeholder = __( 'Start writing…' );
```

Comment:
I wonder if it makes sense to have these placeholder text values as a constant somewhere. I see they are already "hardcoded" in [the implementation](https://github.com/wordpress-mobile/gutenberg-mobile/blob/bd1ebf52f1f9b40880b65ed584c89b9fefcbd6e0/src/components/html-text-input.js#L103). Any changes would probably be for a different PR, but, I'm wondering how likely it is this text will change in the future, and if that will require changes to the utilities so `find` will still return the correct value.

Reply:
Yes, might be! The question here is what is the most appropriate `somewhere` ? :) I sometimes want to do these kind of improvements but end up making things a bit more complicated on JS.
But I'd definitely be willing to update this according to suggestions and what most of the people think about it.

Reply:
> The question here is what is the most appropriate `somewhere`

I agree, this part is something that probably would benefit from input from others, even a quick look 👀 at [examples using Gutenberg block templates API](https://github.com/WordPress/gutenberg/blob/b944ed777b54c8fd93cc894040218550e0fd3c7f/docs/designers-developers/developers/block-api/block-templates.md) gives me the impression that these placeholders are intended to be set inline. I think any decisions / changes around this would not be relevant for this PR, as it seems to be a fairly pervasive pattern that could involve a lot of other areas in the code base.

I only wanted to mention it here to have guard up for something that could break in an unexpected way later on, from a seemingly cosmetic change. It may be that the complexity introduced by having some kind of "static" namespaced constants import (akin to `strings.xml` in Android) would outweigh the benefits derived from having centralized text assets and slightly DRYer code and tests. I'd love to hear others thoughts about it, if anyone has a strong opinion one way or the other.